### PR TITLE
Add material theme to paradox

### PR DESCRIPTION
**Pull request by regadas**
I know #39 is still being worked on but since we have already paradox I thought we could put it to some use in the meantime? wdyt is it worth it?

page

### CODE REVIEW

**Comment by johnynek** [comment]
seems pretty nice to me. Seems pretty easy to merge and change later.

That said, is the site being published?

**Comment by regadas** [comment]
Hi @johnynek, sorry for the late reply! It's not being pushed; I intend to make it so. Just wanted to see if this would be nice to have first. 😄

I'll update this later. Thanks!

**Comment by regadas** [comment]
Hi @johnynek took me longer than expected to follow up on this. Added a step to CI to make the site automatically published on release.

File: project/plugins.sbt [hunk 1]
```diff
@@ -9,3 +9,6 @@ addSbtPlugin("com.codecommit" % "sbt-spiewak-sonatype" % "0.19.1")
 addSbtPlugin("com.codecommit" % "sbt-github-actions" % "0.9.5")
 addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.2")
 addSbtPlugin("org.scalameta" % "sbt-mdoc" % "2.2.13")
+addSbtPlugin("io.github.jonas" % "sbt-paradox-material-theme" % "0.6.0")
+addSbtPlugin("com.typesafe.sbt" % "sbt-site" % "1.4.1")
+addSbtPlugin("com.typesafe.sbt" % "sbt-ghpages" % "0.6.3")
```
Comment (regadas) [inline project/plugins.sbt:14]: These allows us to easily do adhoc site pushes through `docs/ghpagesPushSite`

**Comment by johnynek** [comment]
This is great! I personally think the win of using an all scala/sbt solution is worth it in terms of simplicity of the CI and build.

**Review by johnynek — APPROVED** [review]

**Comment by rossabaker** [comment]
:tada: We use this on http4s-jdk-http-client, and I would love to see it spread.  The multi-version support is lacking, but other than that, it's a solid tool and an attractive, functional theme.  And no futzing with Jekyll or Hugo is a big win.

**Comment by johnynek** [comment]
Related to #7

**Pull request merged**